public/stylesheets/style.css

```diff
@@ -220,6 +234,39 @@ main {
   color: white;
   transform: rotateY(180deg);
 }
+
+.tabs {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  align-items: center;
+}
+.tab h3 {
+  color: white;
+}
+
+.link-container {
```

Comment:
nit pick:
When I see 'link' I assume it's about an `<a>` (anchor). So maybe you could give a bit more descriptive name like `link-preview-[other things]`